src/header/Header.scss

```diff
@@ -1,5 +1,56 @@
+@import "../utils/Mixins";
 @import "../utils/Variables";
 
+@include media-breakpoint-down(sm) {
+  header {
```

Comment:
lets change `header` to `.header`, like in `.section`, so for sure it'll never affect anything other